README.md

```diff
@@ -1 +1,22 @@
 # hw01
+
+I am **Maria Jose Camacho**, second year student at Harris Public Policy School. Before coming to Chicago, I have worked in different agencies in order to define roles, functions and responsibilities at the regulatory, individual and institutional levels to support in the transition of the **criminal justice system** implemented in 20016 in Mexico. 
+
+
+# Work experience
+
+
+## Federal Attorney General’s Office
+
+
+* In my first job after graduation as **federal prosecutor**. I was responsible for defining and establishing the rules required to implement workflow models for the internal processes of each stage within a criminal case under this new legal system. I was also involved in creating criminal investigation protocols, management models for the agency’s **Internal Affairs Office**, and writing legislation that would later become the national guidelines for the new criminal procedure. In addition, I participated in reviewing evidence for *high-impact investigations* involving organized crime and kidnapping. We were tasked with challenging the existing roles and practices associated with the old inquisitorial system.
+
+
+## US Embassy in Mexico City
+
+
+* After leaving my position, I joined the U.S. Embassy in Mexico City under the Bureau of International Narcotics and Law Enforcement Affairs (INL) Section of the U.S. Department of State working directly in the Office of the **Merida Initiative**. My role within INL over the last two years, has included developing and rolling out various projects aimed at strengthening Mexico´s rule of law, as well as working across criminal justice and law enforcement sectors. One of these projects was with **World Justice Project**, an independent organization working to advance the rule of law worldwide. This organization elaborated a report about the current situation of the criminal justice reform in Mexico two years after its implementation. For more information regarding this topic please click the following link: https://worldjusticeproject.org/news/report-new-criminal-justice-system-mexico
```

Comment:
I think this link was a perfect addition. It was well placed, relevant, and enriched your bio overall. I enjoyed checking out the website.

Reply:
This link was super cool and handy, I like how you incorporated so many opportunities for exposure to all the projects you have worked with.

Reply:
Note that you can embed the link in text using the `[]()` syntax.

```
[See here for more information regarding this topic.](https://worldjusticeproject.org/news/report-new-criminal-justice-system-mexico)
```

[See here for more information regarding this topic.](https://worldjusticeproject.org/news/report-new-criminal-justice-system-mexico)

Avoids having to spell out the entire link in the text